src/utils/editor/index.js

```diff
@@ -58,22 +58,18 @@ export function endOperation() {
   codeMirror.endOperation();
 }
 
-export function shouldShowPrettyPrint(selectedSource) {
-  if (!selectedSource) {
-    return false;
-  }
-
-  return shouldPrettyPrint(selectedSource);
+export function shouldShowPrettyPrint(source) {
+  return shouldPrettyPrint(source);
```

Comment:
Now that we are just calling `shouldPrettyPrint`, the additional function, `shouldShowPrettyPrint`. Can't we just use `shouldPrettyPrint`?

Reply:
I kind of like having a function with a presentational name wrapping this.  To me "shouldPrettyPrint" sounds like it determines if an action should take place; "shouldShowPrettyPrint" seems like a good symbol of if something should be presented.

Reply:
Ahan, we can do something like `shouldShowPrettyPrint = shouldPrettyPrint`, that would mean on less call on stack :P